scriptHW5_2.js

```diff
@@ -0,0 +1,52 @@
+const turnButton = document.querySelector(".redBtn");
+const bodyElem = document.querySelector("body");
+const labelElem = document.querySelector(".lastTurn");
+
+if(localStorage.getItem("Date") === null)
+    localStorage.setItem("Date", setCurDate());
+
+if(localStorage.getItem("buttonState") !== null){
+    if(localStorage.getItem("buttonState") == 'Turn on' ){
```

Comment:
```suggestion
    if(localStorage.getItem("buttonState") === 'Turn on' ){
```
Рекомендую ніколи не використовувати не строге порівняння